molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -558,6 +567,36 @@ function($, molgenis, settingsXhr) {
 			});
 		});
 		
+		$('#copy-data-btn').on('click', function(){
+			bootbox.prompt({
+				title: "<h4>Duplicate entity [" + selectedEntityMetaData.label + "]<h4/>" +
```

Comment:
Perhaps "Copy entity" is clearer, since "Duplicate entity" could also mean that the entity name you've entered is already taken.